### ISSUE TITLE
Could not parse POM from jcenter

### ISSUE THREAD

**Issue by paleozogt** [gradle/gradle#5745]
My projects are unable to build due to
```
> Could not resolve org.eclipse.jgit:org.eclipse.jgit:5.0.0.201806131550-r.
 > Could not parse POM https://jcenter.bintray.com/org/eclipse/jgit/org.eclipse.jgit/5.0.0.201806131550-r/org.eclipse.jgit-5.0.0.201806131550-r.pom
    > Could not find org.eclipse.jgit:org.eclipse.jgit-parent:5.0.0.201806131550-r.
```
I can wget/curl the POM directly.  I have also checked the gradle cache and the POM is there.

### Expected Behavior
My project should be able to resolve artifacts on jcenter.

### Current Behavior
```
➜  gradle-bug ./gradlew tasks                                                                                                                                                                    

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'gradle-bug'.
> Could not resolve all files for configuration ':classpath'.
   > Could not resolve org.eclipse.jgit:org.eclipse.jgit:[4.8.0.201706111038-r,).
     Required by:
         project : > org.ajoberstar:grgit:2.0.1
      > Could not resolve org.eclipse.jgit:org.eclipse.jgit:5.0.0.201806131550-r.
         > Could not parse POM https://jcenter.bintray.com/org/eclipse/jgit/org.eclipse.jgit/5.0.0.201806131550-r/org.eclipse.jgit-5.0.0.201806131550-r.pom
            > Could not find org.eclipse.jgit:org.eclipse.jgit-parent:5.0.0.201806131550-r.
              Searched in the following locations:
                  https://jcenter.bintray.com/org/eclipse/jgit/org.eclipse.jgit-parent/5.0.0.201806131550-r/org.eclipse.jgit-parent-5.0.0.201806131550-r.pom
                  https://jcenter.bintray.com/org/eclipse/jgit/org.eclipse.jgit-parent/5.0.0.201806131550-r/org.eclipse.jgit-parent-5.0.0.201806131550-r.jar
   > Could not resolve org.eclipse.jgit:org.eclipse.jgit.ui:[4.8.0.201706111038-r,).
     Required by:
         project : > org.ajoberstar:grgit:2.0.1
      > Could not resolve org.eclipse.jgit:org.eclipse.jgit.ui:5.0.0.201806131550-r.
         > Could not parse POM https://jcenter.bintray.com/org/eclipse/jgit/org.eclipse.jgit.ui/5.0.0.201806131550-r/org.eclipse.jgit.ui-5.0.0.201806131550-r.pom
            > Could not find org.eclipse.jgit:org.eclipse.jgit-parent:5.0.0.201806131550-r.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
See https://docs.gradle.org/4.5.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 1s

➜  gradle-bug head ~/.gradle/caches/modules-2/files-2.1/org.eclipse.jgit/org.eclipse.jgit/5.0.0.201806131550-r/5c14495431e2cbe6e01bceeb976c2b66615930c1/org.eclipse.jgit-5.0.0.201806131550-r.pom
<?xml version="1.0" encoding="UTF-8"?>
<!--
   Copyright (C) 2009, Google Inc.
   Copyright (C) 2009, Igor Fedorenko <igor@ifedorenko.com>
   Copyright (C) 2008, Imran M Yousuf <imyousuf@smartitengineering.com>
   Copyright (C) 2010, Matthias Sohn <matthias.sohn@sap.com>
   and other copyright owners as documented in the project's IP log.

   This program and the accompanying materials are made available
   under the terms of the Eclipse Distribution License v1.0 which

➜  gradle-bug wget https://jcenter.bintray.com/org/eclipse/jgit/org.eclipse.jgit/5.0.0.201806131550-r/org.eclipse.jgit-5.0.0.201806131550-r.pom
--2018-06-20 13:56:01--  https://jcenter.bintray.com/org/eclipse/jgit/org.eclipse.jgit/5.0.0.201806131550-r/org.eclipse.jgit-5.0.0.201806131550-r.pom
Resolving jcenter.bintray.com (jcenter.bintray.com)... 52.42.81.29, 35.160.165.120
Connecting to jcenter.bintray.com (jcenter.bintray.com)|52.42.81.29|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 9461 (9.2K) [application/content-stream]
Saving to: ‘org.eclipse.jgit-5.0.0.201806131550-r.pom’

org.eclipse.jgit-5.0.0.201806131550-r.pom                     100%[==============================================================================================================================================>]   9.24K  --.-KB/s    in 0.001s  

2018-06-20 13:56:01 (16.3 MB/s) - ‘org.eclipse.jgit-5.0.0.201806131550-r.pom’ saved [9461/9461]

➜  gradle-bug head org.eclipse.jgit-5.0.0.201806131550-r.pom                                                                                                                                     
<?xml version="1.0" encoding="UTF-8"?>
<!--
   Copyright (C) 2009, Google Inc.
   Copyright (C) 2009, Igor Fedorenko <igor@ifedorenko.com>
   Copyright (C) 2008, Imran M Yousuf <imyousuf@smartitengineering.com>
   Copyright (C) 2010, Matthias Sohn <matthias.sohn@sap.com>
   and other copyright owners as documented in the project's IP log.

   This program and the accompanying materials are made available
   under the terms of the Eclipse Distribution License v1.0 which

➜  gradle-bug 
```

### Context
I am unable to build my any projects because of this error.

### Steps to Reproduce (for bugs)
Simple build.gradle:
```
buildscript {
    repositories {
        jcenter()
    }
    dependencies {
        classpath 'org.ajoberstar:grgit:2.0.1'
    }
}
```
Run with `./gradlew tasks`

### Your Environment
 * OS: Mac OS X 10.13.5
 * Java 1.8.0_161
 * gradle wrapper with gradle 4.5.1

 * Build scan URL: `./gradlew build --scan` fails with the same POM error.

**Comment by marcphilipp** [comment]
This happens because grgit uses a version range for its JGit dependency and the JGit parent POM (https://jcenter.bintray.com/org/eclipse/jgit/org.eclipse.jgit-parent/5.0.0.201806131550-r/org.eclipse.jgit-parent-5.0.0.201806131550-r.pom) is missing from jcenter. Upgrading to grgit 2.2.1 resolves this issue.

**Comment by paleozogt** [comment]
@marcphilipp This was working up until yesterday... Did something change on the server?

**Comment by marcphilipp** [comment]
Looks like the parent POM linked above is now available on the server again. Can you try again?

**Comment by paleozogt** [comment]
@marcphilipp Ah, that's working now.  I don't understand what happened.  I thought once things were published to a repository like jcenter/maven central they were permanent.

**Comment by marcphilipp** [comment]
They are. However, grgit in version 2.2.0 uses the version range `[4.8.0.201706111038-r,)` for its `org.eclipse.jgit:org.eclipse.jgit` dependency in [its POM](https://plugins.gradle.org/m2/org/ajoberstar/grgit/2.2.0/grgit-2.2.0.pom). That means it will use the latest available version >= `4.8.0.201706111038-r`. Then JGit apparently published an incomplete release (`5.0.0.201806131550-r`). Now they've uploaded the missing artifacts and everything seems to be working again. I think you'd be better of switching to grgit 2.2.1 which uses a fixed version instead, though.
